lib/response-transforms.js

```diff
@@ -32,4 +32,8 @@ module.exports = {
       });
     }
   },
+  jsonify: res => {
```

Comment:
Ok but still... I don't want this to become part of the public API, so can we just add this in the file where it's needed and not to the global HTTP response manipulation functions.